### ISSUE TITLE
using the chinese-pdf-recognition with english

### ISSUE THREAD

**Issue by argosang** [MaybeShewill-CV/CRNN_Tensorflow#267]
Hi I tried out your chinese pdf recognition demo with the english char/ord dict json, but it seems to not understand english letters correctly.  numbers work great. Is there something else I need to do for english to be recognized correctly using your pdf demo? I'm using your pretrained weights from the download link. Below is the example image I am passing in and the resulting output.

![5](https://user-images.githubusercontent.com/47573937/58175563-68eb2b00-7c6e-11e9-84b2-773e41501fb3.png)
`INFO:tensorflow:Restoring parameters from ../weights/shadownet.ckpt
I0522 08:39:46.111400 4257 saver.py:1270] Restoring parameters from ../weights/shadownet.ckpt
----     dolvnurbed
---- 789600babs`

**Comment by MaybeShewill-CV** [comment]
@argosang The simple demo can not be directly used for pdf recognition. You may crop the text area in the pdf by yourself and test if the model can predict the crpped area:)

**Comment by argosang** [comment]
@MaybeShewill-CV The image above that I attached is what I used to test.  The policy number 78960034 image that is.  That's been cropped out of a pdf into a png.

**Comment by MaybeShewill-CV** [comment]
@argosang The model is trained with single line sequence if you're familiar with the Synth90k dataset and the crnn model:)

**Comment by MaybeShewill-CV** [comment]
@argosang Simple demo here 
![2](https://user-images.githubusercontent.com/15725187/58177358-d3d04c80-7cd6-11e9-9a93-949e242bf113.png)

**Comment by argosang** [comment]
Hmm. The pdf demo cuts rows into single lines for processing however, does it not? Ok I will use the test_shadownet.py.  Thanks!

Also is there a way to detect spaces and special characters?  I noticed the char_dict doesn't have spaces and special characters.  How do I add that to the char_dict?

Do you know of a way to do line-detection/segmentation? I will do research on this now.

**Comment by MaybeShewill-CV** [comment]
@argosang If you read the pdf demo code carefully you'll find the single line has been cropped out. The model can predict sequence with blanks. But there is no such labels in synth90k dataset. As for line-detection/segmentation I believe you want to do some research on text area detection? :)
